backend/src/main/java/com/bakdata/conquery/io/storage/xodus/stores/SerializingStore.java

```diff
@@ -397,14 +403,29 @@ public void inject(Injectable injectable) {
 
 	@Override
 	public String toString() {
-		return storeInfo.getXodusName() + "(" + storeInfo.getValueType().getSimpleName() + ")";
+		return storeInfo.getName() + "(" + storeInfo.getValueType().getSimpleName() + ")";
```

Comment:
kannst du hier draus nicht ein `@ToString(of ="storeInfo")` machen, und der StoreInfo auch eine toString zu geben?